src/node_report.cc

```diff
@@ -602,17 +602,22 @@ void PrintNativeStack(FILE* fp) {
         IMAGEHLP_LINE64 line;
         line.SizeOfStruct = sizeof(line);
         if (SymGetLineFromAddr64(hProcess, dwAddress, &dwOffset, &line)) {
-          fprintf(fp, "%2d: [pc=0x%p] %s [+%d] in %s: line: %lu\n", i, pSymbol->Address, pSymbol->Name, dwOffset, line.FileName, line.LineNumber);
+          fprintf(fp, "%2d: [pc=0x%p] %s [+%d] in %s: line: %lu\n", i, 
+			reinterpret_cast<void*>(pSymbol->Address), pSymbol->Name, 
+			dwOffset, line.FileName, line.LineNumber);
         } else {
           // SymGetLineFromAddr64() failed, just print the address and symbol
           if (dwOffset64 <= 32) { // sanity check
```

Comment:
@rnchamberlain Is this sanity check valid? It seems to be consistently failing for me and we drop into the else clause.